WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersCard.swift

```diff
@@ -0,0 +1,88 @@
+
+import SwiftUI
+
+
+struct BloggingRemindersCard: View {
+
+    var getStartedButtonAction: () -> Void
+    var ellipsisButtonAction: () -> Void
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text(TextContent.cardTitle)
+                    .font(TextContent.cardTitleFont)
+                Spacer()
+                Button(action: {
+                    ellipsisButtonAction()
+                }) {
+                    Image(Images.ellipsisImageName)
+                        .foregroundColor(.black)
+                }
+
+            }
+            HStack {
+                VStack {
+                    Image(Images.calendarImageName)
+                        .padding(Metrics.calendarImageInsets)
+                    Spacer()
+                }
+                Spacer()
+                VStack(alignment: .leading) {
+                    Text(TextContent.cardDescription)
+                        .font(TextContent.cardDescriptionFont)
+                        .lineLimit(nil)
+                        .padding()
+                    Spacer()
+                    Button(action: {
```

Comment:
I wonder if we could extract a view small subviews to properties in here just to make it quicker to identify different parts of the view?